src/workflow/EasyAlignproteome.cpp

```diff
@@ -0,0 +1,41 @@
+#include <cassert>
+
+#include "FileUtil.h"
+#include "CommandCaller.h"
+#include "Util.h"
+#include "Debug.h"
+#include "Parameters.h"
+
+#include "easyalignproteome.sh.h"
+
+void setEasyAlignproteomeDefaults(Parameters *p) {
+    p->proteomeSimThr = 0.9;
+}
+
+// void setEasyAlignproteomeMustPassAlong(Parameters *p){
+
```

Comment:
You need to implement this whenever you use the `createParameterString(..., true)` parameter. Or it won't get passed along.